sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -226,14 +230,31 @@ def _management_request(self, mgmt_msg, op_type):
                     mgmt_msg,
                     constants.READ_OPERATION,
                     op_type=op_type,
-                    status_code_field=b"status-code",
-                    description_fields=b"status-description",
+                    status_code_field=MGMT_STATUS_CODE,
+                    description_fields=MGMT_STATUS_DESC,
                 )
-                status_code = response.application_properties[b"status-code"]
+                status_code = response.application_properties[MGMT_STATUS_CODE]
```

Comment:
nit: do we need to cast type here? like int(`response.application_properties[MGMT_STATUS_CODE])`?